src/config/jest.config.js

```diff
@@ -15,8 +15,9 @@ const ignores = [
 const jestConfig = {
   roots: [fromRoot('src')],
   testEnvironment: ifAnyDep(['webpack', 'rollup', 'react'], 'jsdom', 'node'),
-  collectCoverageFrom: ['src/**/*.js'],
-  testMatch: ['**/__tests__/**/*.js'],
+  moduleFileExtensions: ['js', 'jsx', 'json', 'ts', 'tsx'],
+  collectCoverageFrom: ['src/**/*.js', 'src/**/*.js'],
```

Comment:
Should this be:

```
collectCoverageFrom: ['src/**/*.+(js|jsx|ts|tsx)'],
```

Reply:
Also a good question. I got this from a couple projects I work on, which came from else where, and not sure why `jsx` wasn't added. Will get to the bottom of this. Thanks!